docs/Security/Extended-Protection.md

```diff
@@ -282,10 +342,28 @@ If you want to enable Extended Protection in your environment manually without u
 
 ## Known issues and workarounds
 
-1. Customers using a _Retention Policy_ containing _Retention Tags_ which perform _Move to Archive_ actions should not configure Extended Protection, as enabling Extended Protection will cause automated archiving to stop working. We are actively working to resolve this issue.
-2. In Exchange Server 2013, one of the MAPI over HTTP probes (OutlookMapiHttpCtpProbe, used to check if the MAPI/HTTP endpoint is working on the Mailbox server) can show false _FAILED_ status after enabling Extended Protection.
+1. Customers using a _Retention Policy_ containing _Retention Tags_ which perform _Move to Archive_ can now configure Extended Protection with this update. We are actively working on a permanent solution to resolve this issue. Once we ship the solution you will be required to run this script again and rollback the changes.
 
-   For example: running Test-OutlookConnectivity -ProbeIdentity OutlookMapiHttp.Protocol\OutlookMapiHttpCtpProbe may show the probe as failed, although Outlook functionality is working properly.
+2. In Exchange Server 2013, 2016 and 2019 the following probes will show _FAILED_ status after running the script which switches on Extended Protection with required SSL flags on various vDirs as per recommended guidelines:
+   1. OutlookMapiHttpCtpProbe
+   2. OutlookRpcCtpProbe
+   3. OutlookRpcDeepTestProbe
+   4. OutlookRpcSelfTestProbe
+   5. ComplianceOutlookLogonToArchiveMapiHttpCtpProbe
+   6. ComplianceOutlookLogonToArchiveRpcCtpProbe
+
+You will also notice that some Health Mailbox logins fail with event ID: 4625 and failure reason "_An Error occurred during Logon_" and status _0xC000035B_ which is related to the failed probes. [**Get-ServerHealth**](https://docs.microsoft.com/en-us/exchange/high-availability/managed-availability/health-sets?view=exchserver-2019#use-the-exchange-management-shell-to-view-a-list-of-monitors-and-their-current-health) command will also show RPC and Mapi monitors as Unhealthy.
+
```

Comment:
I’ve added the PS commands to the doc (in review mode). We should add them here. To be confirmed by @mukeshHCW